Gems/PhysX/Code/Editor/Source/ComponentModes/Joints/JointsComponentModeCommon.cpp

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <Editor/Source/ComponentModes/Joints/JointsComponentModeCommon.h>
+
+namespace PhysX::JointsComponentModeCommon
+{
+    const AZStd::string ParamaterNames::TwistLimits = "Twist Limits";
+    const AZStd::string ParamaterNames::Damping = "Damping";
+    const AZStd::string ParamaterNames::MaxForce = "Maximum Force";
+    const AZStd::string ParamaterNames::MaxTorque = "Maximum Torque";
+    const AZStd::string ParamaterNames::Position = "Position";
+    const AZStd::string ParamaterNames::Rotation = "Rotation";
+    const AZStd::string ParamaterNames::SnapPosition = "Snap Position";
+    const AZStd::string ParamaterNames::SnapRotation = "Snap Rotation";
+    const AZStd::string ParamaterNames::Stiffness = "Stiffness";
+    const AZStd::string ParamaterNames::SwingLimit = "Swing Limits";
+    const AZStd::string ParamaterNames::Transform = "Transform";
+    const AZStd::string ParamaterNames::ComponentMode = "Component Mode";
+    const AZStd::string ParamaterNames::LeadEntity = "Lead Entity";
```

Comment:
update: Prefer `string_view` for these (or just `const char* const`) - it'll wind up being more efficient under the hood

Reply:
i think i actually just moved this from elsewhere. But i'll update the type (good catch).

Reply:
Nice one thanks!